code/datums/spells/wizard.dm

```diff
@@ -157,6 +157,19 @@
 	cast_sound = "sound/magic/ForceWall.ogg"
 
 
+/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop
+	name = "Stop Time"
+	desc = "This spell stops time for everyone except for you, allowing you to move freely while your enemies and even projectiles are frozen."
+	charge_max = 300
+	clothes_req = 1
+	invocation = "TOKI WO TOMARE"
+	invocation_type = "shout"
+	range = 0
+	summon_amt = 1
+	action_icon_state = "time"
+

```

Comment:
I agree, that sounds fun. Maybe enough that it nearly permastops at four, with like a second or two to react before you get stopped again?